app/Http/Controllers/UserController.php

```diff
@@ -140,37 +142,44 @@ public function SaveAccount(Request $request): RedirectResponse {
         return redirect()->route('account');
     }
 
+    /**
+     * @api v1
+     * @frontend
+     * @param User $user
+     * @return LengthAwarePaginator|null
+     */
+    public static function statusesForUser(User $user): ?LengthAwarePaginator {
+        if ($user->userInvisibleToMe) {
+            return null;
+        }
+        return $user->statuses()->with('user',
+                                       'trainCheckin',
+                                       'trainCheckin.Origin',
+                                       'trainCheckin.Destination',
+                                       'trainCheckin.HafasTrip',
+                                       'event')->orderBy('created_at', 'DESC')->paginate(15);
```

Comment:
```suggestion
                                       'event')->orderByDesc('created_at')->paginate(15);
```

(Please check before commit)